src/node/glossary.ts

```diff
@@ -0,0 +1,12 @@
+import { SharedOptions } from '../sharedOptions'
+import { RequestHandlersList } from '../setupWorker/glossary'
+
+export type ListenOptions = SharedOptions
```

Comment:
We can remove the `ListenOptions` type as it's effectively a copy of the `SharedOptions`. Let's use the latter directly, unless we have some extra types to union with the shared options.

```ts
listen(options?: SharedOptions): void
```